DEVELOPMENT.md

```diff
@@ -115,3 +139,49 @@ bazel build third_party/chromium
 
 # Lastly, put the address returnd by the web server into the Chromium.
 ```
+
+#### Running webapp tests
+
+tl;dr. Use `ibazel test` for regular work (supports limiting the test amount with
+`fit/fdescript` and `console.log` usage) and `ibazel run` for karma console breakpoint
+debugging.
+
+We use karma testing chromium target for our UI unit tests in webapp. However,
+depending on the environments (OSS or internal TB.corp), the commands we use and
+the functionalities are slightly different.
+
+1.  Just run all webapp tests. The job stops after finished. `console.log` is not
+    supported. Not handy on development.
```

Comment:
"Not handy on development": I use it all the time for development :). Do you mean it's not useful for debugging?